geniza/corpus/management/commands/export_metadata.py

```diff
@@ -101,35 +115,49 @@ def paths(self):
     ## LOCAL EXPORTING
     ############################################
 
-    def export_data(self):
+    def export_data(self, lastrun, sync=False):
         "generate all exports"
-        with self.timer("Exporting metadata into local path"):
-            # make sure to pull first
-            self.repo_pull()
-
-            # write docs
-            with self.timer("Exporting document data"):
-                PublicDocumentExporter(progress=self.progress).write_export_data_csv(
-                    self.path_documents_csv
-                )
-
-            # write fragments
-            with self.timer("Exporting fragment data"):
-                PublicFragmentExporter(progress=self.progress).write_export_data_csv(
-                    self.path_fragments_csv
-                )
-
-            # write sources
-            with self.timer("Exporting source data"):
-                SourceExporter(progress=self.progress).write_export_data_csv(
-                    self.path_sources_csv
-                )
-
-            # write footnotes
-            with self.timer("Exporting footnotes"):
-                FootnoteExporter(progress=self.progress).write_export_data_csv(
-                    self.path_footnotes_csv
-                )
+
+        # pull any remote changes before making any local modifications
+        self.repo_pull()
+
+        # get all log entries since the last run
+        log_entries = LogEntry.objects.filter(action_time__gte=lastrun)
+
+        # if there are NO changes, bail out
+        if log_entries.count() == 0:
+            self.print("No changes since last run")
+            return
```

Comment:
Good, it's nice we don't have to actually run the export when nothing changes (and the cron job will exit swiftly)